src/explore-education-statistics-common/src/components/AnchorHtml.tsx

```diff
@@ -0,0 +1,40 @@
+import { OmitStrict } from '@common/types';
+import applyRedirectRules from '@common/utils/url/applyRedirectRules';
+import React, { AnchorHTMLAttributes, useEffect, useState } from 'react';
+
+interface AnchorHtmlProps
+  extends OmitStrict<AnchorHTMLAttributes<HTMLAnchorElement>, 'href'> {
+  href: string;
+}
+
+export default function AnchorHtml({
```

Comment:
Would just call this `Link`, or `ContentLink` if you want to avoid confusion with the other `Link` components.